### PR TITLE
remove some of the net48 configs

### PR DESCRIPTION
**Pull request by pinzart90**
Remove net48 configs and PR checks

### CODE REVIEW

File: src/buildNET6.xml [hunk 1]
```diff
@@ -1,25 +0,0 @@
...
-    ToolsVersion="15.0"
-    DefaultTargets="Build">
-<PropertyGroup>
-    <SolutionNET6>Dynamo.All.Net6.sln</SolutionNET6>
-    <NuGetPath>$(MSBuildProjectDirectory)\..\tools\Nuget\NuGet.exe</NuGetPath>
- </PropertyGroup>
-	
-<ItemGroup>
-    <ProjectToBuild Include="$(SolutionNET6)" >
-        <Properties>Configuration=Release;Platform=NET60_Windows</Properties>
```
Comment (mjkkirschner) [inline src/buildNET6.xml:12]: I think removing this file may break the master-15 builds... @QilongTang does that sound right?

File: .github/workflows/dynamoBinDiff.yml [hunk 2]
```diff
@@ -1,197 +0,0 @@
-# Build Dynamo using latest VS and DotNET and perform a Bin Diff
```
[the remark this answers is not in the document]
Comment (pinzart90) [reply .github/workflows/dynamoBinDiff.yml:1]: there s a copy of it on 2.19 
Why is it valuable on master ?

File: .github/workflows/dynamoAllNet6.0_build.yml [hunk 3]
```diff
@@ -18,12 +18,12 @@ jobs:
...
 
     - name: Install dependencies for windows runtime
-      run: dotnet restore $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.Net6.sln -p:Platform=NET60_Windows --runtime=win10-x64
+      run: dotnet restore $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln -p:Platform=NET60_Windows --runtime=win10-x64
     - name: Build Dynamo with MSBuild for Windows
       run: |
         echo "***Continue with the build, Good luck developer!***"
         cd "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\MSBuild\Current\Bin\"
-           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.Net6.sln /p:Configuration=Release  /p:Platform=NET60_Windows
+           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln /p:Configuration=Release  /p:Platform=NET60_Windows
```
Comment (aparajit-pratap) [inline .github/workflows/dynamoAllNet6.0_build.yml:26]: Do you want to rename this yml file too to remove the "net6.0" from the name?

Comment (pinzart90) [reply .github/workflows/dynamoAllNet6.0_build.yml:26]: Well the yaml is suggestive that we are building based on net6 (the dotnet framework)
Maybe that is somewhat helpful
if not I can rename it...
I can imagine another yaml for net8 (dynamoAllNet8.0_build)

File: src/buildNET6.xml (hunk 1 above)
Comment (pinzart90) [reply src/buildNET6.xml:12]: Master 15 will probably need to implement some conditionals based on the branch name
if master => no more net6 solutions or config
if 2,19 branch => as before

File: .github/workflows/dynamoBinDiff.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mjkkirschner) [reply .github/workflows/dynamoBinDiff.yml:1]: going forward I still think it's useful for devs to understand what new binary changes they are making in their pr and it makes review of that easier.

[the remark this answers is not in the document]
Comment (aparajit-pratap) [reply .github/workflows/dynamoBinDiff.yml:1]: I agree with @mjkkirschner

File: src/Dynamo.All.sln
Comment (aparajit-pratap) [inline src/Dynamo.All.sln:1]: So this is basically the Dynamo.All.Net6 solution renamed to Dynamo.All?

File: src/buildNET6.xml (hunk 1 above)
Comment (QilongTang) [reply src/buildNET6.xml:12]: Removing the file may break but replacing is probably fine. The only part concerning me is that the build path is totally different from before, this change does not worry me for now

File: src/Dynamo.All.sln
Comment (pinzart90) [reply src/Dynamo.All.sln:1]: yes

File: .github/workflows/dynamoBinDiff.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pinzart90) [reply .github/workflows/dynamoBinDiff.yml:1]: Oh, for some reason I though there was another binDiff file for net6
I'll restore it

**Review by mjkkirschner — APPROVED** [review]

File: .github/workflows/dynamoBinDiff.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pinzart90) [reply .github/workflows/dynamoBinDiff.yml]: restored

**Pull request merged**